### PR TITLE
ISPN-11151 - Migrating some remote jdbc-cachestore tests from jdg-functional-tests to upstream

### PR DESCRIPTION
**Pull request by gustavolira**
https://issues.redhat.com/browse/ISPN-11151

### CODE REVIEW

**Comment by tristantarrant** [comment]
This needs a Jira

**Comment by gustavolira** [comment]
> This needs a Jira

Done

File: server/runtime/src/test/java/org/infinispan/server/functional/RestOperations.java [hunk 1]
```diff
@@ -73,6 +74,19 @@ public void testRestOperations() {
       assertEquals(protocol, response.getProtocol());
    }
 
+   @Test
+   public void testPutWithTimeToLeave() throws InterruptedException {
```
Comment (tristantarrant) [inline server/runtime/src/test/java/org/infinispan/server/functional/RestOperations.java]: I assume that should be `testPutWithTimeToLive`

File: server/runtime/src/test/java/org/infinispan/server/persistence/JdbcStringBasedCacheStorePassivationEnabled.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+ * preload == false --after server restart, entries should NOT be preloaded to the cache
+ * purge == false --all entries should remain in the cache store after server restart
+ * (must be false so that we can test preload)
+ * <p>
+ * Other attributes like singleton, shared, fetch-state do not make sense in single node cluster.
+ *
+ */
+@Category(Persistence.class)
+@RunWith(Parameterized.class)
+public class JdbcStringBasedCacheStorePassivationEnabled {
```
Comment (tristantarrant) [inline server/runtime/src/test/java/org/infinispan/server/persistence/JdbcStringBasedCacheStorePassivationEnabled.java]: This needs to be added to PersistenceIT

File: server/runtime/src/test/java/org/infinispan/server/persistence/JdbcStringBasedCacheStorePassivationDisabled.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+ * passivation == false --all cache entries should always be also in the cache store
+ * preload == true --after server restart, all entries should appear in the cache immediately
+ * purge == false --all entries should remain in the cache store after server  * restart
+ * <p>
+ * Other attributes like singleton, shared, fetch-state do not make sense in single node cluster.
+ *
+ */
+@Category(Persistence.class)
+@RunWith(Parameterized.class)
+public class JdbcStringBasedCacheStorePassivationDisabled {
```
Comment (tristantarrant) [inline server/runtime/src/test/java/org/infinispan/server/persistence/JdbcStringBasedCacheStorePassivationDisabled.java]: Add to PersistenceIT

**Comment by tristantarrant** [comment]
Closing for inactivity. Please reopen once ready.

**Pull request closed without merging**